dev/src/Domain/ProductInOrder.java

```diff
@@ -37,10 +41,31 @@ public int amount() {
         return amount;
     }
 
+
+    // Gets also the amount to check if there is a discount for the checked supplier
+    public static Supplier getCheapestSupplier(String product, int amount, List<Supplier> suppliers){
```

Comment:
why is it static? it recives a product and amount, those can be found in ProductInOrder., Should be a class method

Reply:
The IntelliJ required it since the calling function (getFromIO) is static so it's cannot call local function,
if you know how to make it work you can show me pls 🦩